helm/cluster-gcp/templates/_gcp_cluster.tpl

```diff
@@ -2,6 +2,8 @@
 apiVersion: infrastructure.cluster.x-k8s.io/v1beta1
 kind: GCPCluster
 metadata:
+  annotations:
+    bastion.gcp.giantswarm.io/whitelist: {{ .Values.bastion.whitelistSubnets }}
```

Comment:
What would be consuming this annotation?

Reply:
`capg-firewall-rule-operator`